NitroxModel/DataStructures/GameLogic/Entities/VehicleEntity.cs

```diff
@@ -18,20 +18,21 @@ public class VehicleWorldEntity : GlobalRootEntity
     public float ConstructionTime { get; set; }
 
     [IgnoreConstructor]
-    protected VehicleWorldEntity()
+    protected VehicleEntity()
     {
         // Constructor for serialization. Has to be "protected" for json serialization.
     }
 
-    public VehicleWorldEntity(NitroxId spawnerId, float constructionTime, NitroxTransform transform, string classId, bool spawnedByServer, NitroxId id, NitroxTechType techType, EntityMetadata metadata) :
-        base(transform, 0, classId, spawnedByServer, id, techType, metadata, null, new List<Entity>())
+    // 100 is GlobalRoot level
```

Comment:
Is it worth us having this in a constant somewhere so all things that need the globalroot can use the same and it's not magic numbers? Just a thought, I think the comment also helps